certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -201,6 +201,64 @@
         "x-ms-no-generic-test": true
       }
     },
+    "/accounts/{accountId}/connectV2/{connectId}": {
+      "delete": {
```

Comment:
Does this PR have R2 work that you are trying to merge to the release branch?

Reply:
Yes, it has both R2 changes and C2 changes.
Since there are overlapping requirements and also they are being shipped in same time frame, I was thinking to just have a common branch rather than separate.